src/cargo/core/compiler/mod.rs

```diff
@@ -1265,8 +1293,13 @@ fn on_stderr_line_inner(
             #[derive(serde::Deserialize)]
             struct CompilerMessage {
                 rendered: String,
+                level: String,
             }
             if let Ok(mut error) = serde_json::from_str::<CompilerMessage>(compiler_message.get()) {
+                if error.level == "error" && error.rendered.contains("aborting due to") {
```

Comment:
Would it be possible to add the `message` field to `CompilerMessage`, and compare against that instead of `rendered`?  I would feel more comfortable than comparing against the `rendered` string. Although unlikely, it seems possible for it to get a false positive (for example, if the user's code contains those words).

Reply:
Sure thing, done :) and I changed it to `starts_with` as well to make it even more unlikely it will collide.